src/pages/home/library/EditSong.tsx

```diff
@@ -0,0 +1,120 @@
+import { Box, Stack, Tab, Tabs, Theme, Typography } from "@mui/material";
+import { ReactNode, SyntheticEvent, useState } from "react";
+import { useLocation, useNavigate } from "react-router";
+import ArrowLeft from "assets/images/ArrowLeft";
+import VerticalEllipsis from "assets/images/VerticalEllipsis";
+import theme from "theme";
+import { Button } from "elements";
+import { ProfileImage } from "components";
+import { Song } from "modules/song";
+
+interface TabPanelProps {
+  children: ReactNode;
+  index: number;
+  value: number;
+}
+
+interface ColorMap {
+  [index: number]: Partial<keyof Theme["gradients" | "colors"]>;
+}
+
+const TabPanel = ({ children, value, index }: TabPanelProps) => {
```

Comment:
I could extract out the Tabs/TabPanel into its own component if you all feel that we will have a good use in the future?